execute/executor.go

```diff
@@ -132,12 +126,41 @@ type createExecutionNodeVisitor struct {
 	nodes map[plan.PlanNode]Node
 }
 
+func skipYields(pn plan.PlanNode) plan.PlanNode {
+	isYield := func(pn plan.PlanNode) bool {
+		_, ok := pn.ProcedureSpec().(plan.YieldProcedureSpec)
```

Comment:
You mean to put the `YieldProcedureSpec` interface in `transformations/yield.go`?  I think it still belongs in the `plan` package to avoid issues with cyclical dependencies.  I don't think `execute` can depend on `transformations`.

Reply:
Ah yes you're right. Never mind.